membership-attribute-service/app/models/Attributes.scala

```diff
@@ -57,10 +60,55 @@ object Attributes {
     (__ \ "adFree").writeNullable[Boolean] and
     (__ \ "cardExpirationMonth").writeNullable[Int] and
     (__ \ "cardExpirationYear").writeNullable[Int] and
-    (__ \ "contributionFrequency").writeNullable[String] and
-      (__ \ "membershipJoinDate").writeNullable[LocalDate]
+    (__ \ "contributionPaymentPlan").writeNullable[String] and
+    (__ \ "membershipJoinDate").writeNullable[LocalDate]
   )(unlift(Attributes.unapply)).addField("contentAccess", _.contentAccess)
 
+
+
   implicit def toResult(attrs: Attributes): Result =
     Ok(Json.toJson(attrs))
 }
+
+
+
+case class MembershipAttributes(
+                       UserId: String,
+                       Tier: Option[String] = None,
+                       MembershipNumber: Option[String],
+                       AdFree: Option[Boolean] = None,
+                       ContentAccess : MembershipContentAccess)
+
+object MembershipAttributes {
+
+  implicit val jsWrite: OWrites[MembershipAttributes] = (
+    (__ \ "userId").write[String] and
+    (__ \ "tier").writeNullable[String] and
```

Comment:
Also, please check therefore that the /me/membership endpoint does a 404 for a valid recurring contributor.